### DISCUSSION TITLE
AxisX range, MinValue and MaxValue

### DISCUSSION THREAD

**Discussion by kMOsmane** [Live-Charts/LiveCharts2#214] [Ideas]
Hello, I was using theses 2 events in livecharts V1, PreviewRangeChanged and RangeChanged, to modify the range of my X Axis (min and max), but now I didn't find it in the Livecharts V2 , so how I can change the min and max value of my X Axis in this new version ?

thank you.

**Comment by preintel** [comment]
Check out the source code of the file Axis.cs (can download the src folder from github), code is below. I am not sure about the events triggering but you can easily set the axis property for min and max values when creating the axis:

 ```
myViewModelClass.YAxes = new[]
                                      {
                                                                    new Axis
                                                                    {
                                                                        Name = "Price",
                                                                        Labeler = value => "$"+value.ToString(),
                                                                        MaxLimit = myMaxLimitDouble,
                                                                        MinLimit = myMinLimitDouble
                                                                    }
                                                                };
```

https://github.com/beto-rodriguez/LiveCharts2/blob/master/src/LiveChartsCore/Axis.cs

**Comment by beto-rodriguez** [comment]
Use the `PropertyChanged` event, finally check when the property name is `MinLimit` or `MaxLimit`

**Comment by Gossenruda** [comment]
Hi,

I am trying to do this to. @beto-rodriguez on what object should I set the PropertyChanged? I have tried the Axis but can't seem to find MinValue and MaxValue there.

The reason I am asking is that I would like to place two diagrams vertically that share the same intervall.

**Reply by beto-rodriguez** [reply]
@Gossenruda My bad, I meant  `MinLimit` or `MaxLimit`

**Comment by Gossenruda** [comment]
Aha, ok. I think I have managed to solve it by adding an additional series with min and max value as two points and setting the series to be invisible. Maybe this can help @kMOsmane 🙂
